src/dragonball/src/dbs_virtio_devices/src/vhost/vhost_user/test_utils.rs

```diff
@@ -700,6 +700,8 @@ pub(crate) fn negotiate_slave(
             std::mem::size_of::<VhostUserInflight>() as u32,
         );
         slave.send_header(&hdr, None).unwrap();
+    } else{
+        slave.send_header(&hdr, None).unwrap();
```

Comment:
Comments needed to explain that directly using .unwrap() is safe

Reply:
This file is for unit testing and should fail immediately if something goes wrong.